homeassistant/components/suez_water/config_flow.py

```diff
@@ -77,9 +77,10 @@ async def async_step_user(
                 _LOGGER.exception("Unexpected exception")
                 errors["base"] = "unknown"
             else:
-                return self.async_create_entry(
-                    title=user_input[CONF_USERNAME], data=user_input
-                )
+                meter_id = str(user_input[CONF_COUNTER_ID])
```

Comment:
Already a string?

Sometimes it's called `meter_id` and sometimes `counter_id`...

Reply:
See above for string, I'll check.

Counter / meter 😅 my fault I'm planning to change everything because I understood that my counter isn't very English, but I try to keep myself with the old counter